microsoft-edge/progressive-web-apps-chromium/how-to/push.md

```diff
@@ -6,155 +6,141 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: pwa
-ms.date: 06/16/2023
+ms.date: 08/03/2023
 ---
 # Re-engage users with push messages
 
 Push messages are a useful communication channel that lets applications update their users with relevant and timely content. Push messages can be used to re-engage users with your app.
 
 One of the most significant advantages of push messages is that they can be delivered by your app's server even when the user isn't actively using your app.
 
-Push message notifications are useful for apps to take part in the system's notification center and display images and text information.  Notifications are useful to alert the user about an important updates in your app.  However, notifications should be used rarely, because they tend to be disruptive to the user's workflow.
+Push message notifications take part in the system's notification center display images and text information.  Notifications are useful to alert the user about an important updates in your app.  However, notifications should be used rarely, because they tend to be disruptive to the user's workflow.
```

Comment:
```suggestion
Push message notifications take part in the system's notification center, and they can display images and text information.  Notifications are useful to alert the user about an important updates in your app.  However, notifications should be used rarely, because they tend to be disruptive to the user's workflow.
```
* Restored 'and', to be grammatical.
* Comma to try to clarify meaning.

Trying to interpret & clarify orig. sentence by expanding it:
Push message notifications are useful for apps to take part in the system's notification center.  Push message notifications are also useful for apps to and display images and text information.